src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/StopPoints/Editor/index.tsx

```diff
@@ -81,41 +73,135 @@ const StopPointEditor = (props: Props & StateProps) => {
+            setSelectMode(e.target.value as StopPlaceMode);
+            setQuaySearch(undefined);
+            stopPointChange({
+              ...stopPoint,
+              quayRef: undefined,
+              flexibleStopPlaceRef: undefined,
+              flexibleStopPlace: undefined
+            });
+          }}
+        >
+          <Radio value="custom">{formatMessage(messages.selectCustom)}</Radio>
+          <Radio value="nsr">{formatMessage(messages.selectNSR)}</Radio>
+        </RadioGroup>
+      </div>
+      <div className="stop-point-info">
+        {selectMode === 'custom' && (
+          <Dropdown
+            label={formatMessage(messages.stopPlace)}
+            value={
+              stopPoint.flexibleStopPlaceRef ?? stopPoint.flexibleStopPlace?.id
+            }
+            items={flexibleStopPlaces.map(fsp => ({
+              value: fsp.id,
+              label: fsp.name ?? ''
+            }))}
+            onChange={e =>
+              stopPointChange({ ...stopPoint, flexibleStopPlaceRef: e?.value })
+            }
+            variant={stopPlaceError ? 'error' : undefined}
+            feedback={
+              stopPlaceError ? formatMessage(stopPlaceError) : undefined
+            }
+          />
+        )}
+
+        {selectMode === 'nsr' && (
+          <InputGroup
+            label={formatMessage(messages.labelQuayRef)}
+            feedback={
+              stopPlaceError ? formatMessage(stopPlaceError) : undefined
+            }
+            variant={stopPlaceError ? 'error' : undefined}
+            {...quaySearchResults(quaySearch)}
+          >
+            <TextField
+              defaultValue={stopPoint.quayRef ?? ''}
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                debouncedSearchForQuay(e.target.value)
+              }
+              onBlur={(e: ChangeEvent<HTMLInputElement>) =>
+                stopPointChange({ ...stopPoint, quayRef: e.target.value })
+              }
+            />
+          </InputGroup>
+        )}
+
+        <InputGroup
+          label={`${frontTextRequired ? '* ' : ''}${formatMessage(
+            messages.labelFrontText
+          )}`}
+          variant={frontTextError ? 'error' : undefined}
+          feedback={frontTextError ? formatMessage(frontTextError) : undefined}
+        >
+          <TextField
+            defaultValue={stopPoint.destinationDisplay?.frontText ?? ''}
+            onChange={(e: ChangeEvent<HTMLInputElement>) =>
+              stopPointChange({
+                ...stopPoint,
+                destinationDisplay: { frontText: e.target.value }
+              })
+            }
+          />
+        </InputGroup>
+        <Dropdown
+          label={formatMessage(messages.labelBoarding)}
+          value={convertBoardingToDropdown(stopPoint)}
+          onChange={value =>
+            stopPointChange({
+              ...stopPoint,
+              forBoarding: value?.value === '0' || value?.value === '2',
```

Comment:
value?.value 😄

Reply:
Hehe, kan kanskje kalle den ytre value for noe annet ja 😛